modules/lite/src/index.js

```diff
@@ -1,12 +1,12 @@
 /* global window */
 import * as lumaGL from 'luma.gl';
```

Comment:
[DISCUSSION MODE]
- Since people are not going to actually be 'importing' this file, it is not an `index.js` I'd suggest this file being called `bundle.js`, and people pick up the result from `node_modules/@deck.gl/lite/bundle.js`.

What is missing is an index file with the ability to import the `Mapbox` wrapper into standard JS deck.gl apps.